projects/test-statistics-app/src/lib/time-series-chart.ts

```diff
@@ -0,0 +1,171 @@
+import {Component, ElementRef, Input, OnChanges, ViewRef} from '@angular/core';
+
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import {default as vegaEmbed} from 'vega-embed';
+
+import {
+  HsDialogContainerService,
+  HsLanguageService,
+  HsLayerUtilsService,
+} from 'hslayers-ng';
+
+dayjs.extend(utc);
+const CHART_DIV = '.hs-statistics-timeseries';
+@Component({
+  selector: 'hs-time-series-chart',
+  templateUrl: './time-series-chart.component.html',
+})
+export class HsStatisticsTimeSeriesChartComponent implements OnChanges {
+  @Input() observations: {
```

Comment:
This input always is undefined when I tried to open prediction or time series charts.

Reply:
Yes, observations is filled later when predictions and input variables by year are filled.

Reply:
> > Also error Expression has changed after it was checked about -> `[ngClass]="{'hs-open': HsLayoutService.apps[app].sidebarExpanded && sidebarVisible, 'hs-sb-right': sidebarPosition === 'right' && sidebarVisible, 'hs-sb-left': sidebarPosition === 'left' && sidebarVisible,'hs-sb-bottom': sidebarPosition === 'bottom'}">`
> > Also error -> styles.js:3336 Uncaught SyntaxError: Cannot use 'import.meta' outside a module (at styles.js:3336:29)
> 
> interesting. When does that appear?

Well instantly, when the app gets loaded.

Reply:
Well I get error about it, that this.observations is not iterable, well because it is undefined.

Reply:
> Well I get error about it, that this.observations is not iterable, well because it is undefined.

Should be fine now